test/src/go.mod

```diff
@@ -1,8 +1,11 @@
-module github.com/champ-oss/terraform-module-template
+module github.com/champtitles/reponame
```

Comment:
this should be github.com/champ-oss/terraform-mongo-atlas

Reply:
good catch, updated